test/crypto_sign/functest.c

```diff
@@ -5,6 +5,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <valgrind/memcheck.h>
```

Comment:
Can we hide the Valgrind tests behind a `#define`? I think `PQCLEAN_USE_VALGRIND` or something along those lines sounds good. 

Then we can pass this define in `test_valgrind.py`.

Reply:
Done! Also rebased on the latest `master`, so hopefully CI will be green(er).